src/Nethermind/Nethermind.Merge.Plugin/MergePlugin.cs

```diff
@@ -344,11 +344,18 @@ public Task InitSynchronization()
                     _api.LogManager);
 
                 _api.UnclesValidator = new MergeUnclesValidator(_poSSwitcher, _api.UnclesValidator);
-                _api.BlockValidator = new BlockValidator(_api.TxValidator, _api.HeaderValidator, _api.UnclesValidator,
-                    _api.SpecProvider, _api.LogManager);
+                _api.BlockValidator = new InvalidBlockInterceptor(
```

Comment:
Not related to this PR, but I noticed that we have _api.BlockValidator in two places

Reply:
InitNetworkProtocol https://github.com/NethermindEth/nethermind/blob/master/src/Nethermind/Nethermind.Merge.Plugin/MergePlugin.cs#L200 and InitSynchronization?

Reply:
Yes, which is why it was missed.

Reply:
Yes, which is why it was missed.

Reply:
Yes, which is why it was missed.